v1/feed.yaml

```diff
@@ -0,0 +1,105 @@
+# Feed endpoints powered by the MCS
+
+swagger: 2.0
+paths:
+  /featured/{yyyy}/{mm}/{dd}:
+    get:
+      tags:
+        - Feed
+      summary: Aggregated featured content
+      description: |
+        Provides the aggregated feed content for the given date. The endpoint returns
+        the featured article of the day, most read articles for the previous day, news
+        content, a random article, and the featured image and video of the day.
+
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: yyyy
+          in: path
+          description: 'Year the aggregated content is requested for'
+          type: string
+          required: true
+        - name: mm
+          in: path
+          description: 'Month the aggregated content is requested for'
+          type: string
+          required: true
+        - name: dd
+          in: path
+          description: 'Day of the month the aggregated content is requested for'
+          type: string
+          required: true
+      responses:
+        '200':
+          description: JSON containing all of the featured content
+          schema:
+            type: object
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-request-handler:
+        - from_mobileapps:
+            request:
+              method: get
+              uri: '{{options.host}}/{domain}/v1/feed/featured/{yyyy}/{mm}/{dd}'
```

Comment:
There are two ways to go about this: have an SLA between MCS and RB for the format from which RB will assemble the content, or have MCS return the whole aggregated result and then store each piece separately (as this endpoint aggregates multiple ones), just as we do for `mobile-sections`. This version doesn't store anything, it merely exposes publicly the the endpoint and its format.

We are still free to go either way independently of this PR. I'm not sure which of the two options is better, to be honest, because of the `random` component in the response...